projects/angular-mydatepicker/src/lib/components/day-view/day-view.component.html

```diff
@@ -16,11 +16,17 @@
                 'myDpSelectedDay':!this.opts.dateRange && isDateSame(d.dateObj) || this.opts.dateRange && isDateRangeBeginOrEndSame(d.dateObj),
                 'myDpDisabled': d.disabled,
                 'myDpTableSingleDay': !d.disabled}"
-          (click)="onDayCellClicked($event, d)" (keydown)="onDayCellKeyDown($event, d)"
-          (mouseenter)="onDayCellMouseEnter(d)" (mouseleave)="onDayCellMouseLeave()" tabindex="0">
-        <span *ngIf="d.markedDate.marked" class="myDpMarkDate" [ngStyle]="{'background-color': d.markedDate.color}"></span>
-        <span  class="myDpDayValue" [ngClass]="{'myDpMarkCurrDay': d.currDay && opts.markCurrentDay, 'myDpDimDay': d.highlight && (d.cmo===prevMonthId || d.cmo===nextMonthId || d.disabled), 'myDpHighlight': d.highlight}">{{d.dateObj.day}}</span>
-      </td>
-    </tr>
+        (click)="onDayCellClicked($event, d)"
+        (keydown)="onDayCellKeyDown($event, d)"
+        (mouseenter)="onDayCellMouseEnter(d)"
+        (mouseleave)="onDayCellMouseLeave()"
+        tabindex="0">
+      <span *ngIf="d.markedDate.marked" class="myDpMarkDate" [ngStyle]="{'background-color': d.markedDate.color}"></span>
+      <span class="myDpDayValue"
+          [ngClass]="{'myDpMarkCurrDay': d.currDay && opts.markCurrentDay, 'myDpDimDay': d.highlight && (d.cmo===prevMonthId || d.cmo===nextMonthId || d.disabled), 'myDpHighlight': d.highlight}"
+          [ngStyle]="{'background-color': d.color}">{{d.dateObj.day}}
```

Comment:
using `class` instead of color would be a prettier solution

Reply:
using ngClass instead of ngStyle?